lib/sedna/service/server/aggregation.py

```diff
@@ -266,3 +262,27 @@ async def client_info(self, request: Request):
         if client_id:
             return server.get_client(client_id)
         return WSClientInfoList(clients=server.client_list)
+
+class AggregationServerv2():
```

Comment:
same issue  `AggregationServerv2` as above

Reply:
After discussion, we keep two interfaces in this PR, will update them at the next PR.